controllers/idler/idler_controller_test.go

```diff
@@ -1317,12 +1327,94 @@ func preparePayloadsSinglePod(t *testing.T, r *Reconciler, namespace, namePrefix
 	}
 }
 
-func createPods(t *testing.T, r *Reconciler, owner metav1.Object, startTime metav1.Time, podsToTrack []*corev1.Pod, conditions ...corev1.PodCondition) []*corev1.Pod {
+func preparePayloadCrashloopingPod(t *testing.T, r *Reconciler, namespace, namePrefix string) payloads {
+	standalonePods := make([]*corev1.Pod, 0, 1)
+	startTime := metav1.Now()
+	replicas := int32(3)
+	// Create a standalone pod with no owner which has restart count sum > 50
```

Comment:
Just an update to the comment since it's no longer the sum of restarts right?
```suggestion
	// Create a standalone pod with no owner which has at least one container with restart count > 50
```

Reply:
done in [a147e63](https://github.com/codeready-toolchain/member-operator/pull/630/commits/a147e636496395427a349188b8cd8a91bfc020ea)